wp-sentry.php

```diff
@@ -96,7 +96,7 @@ function wp_sentry_php_version_notice() { ?>
 }
 
 // Load the admin page when needed
-if ( is_admin() ) {
+if ( function_exists( 'is_admin' ) && is_admin() ) {
 	WP_Sentry_Admin_Page::get_instance();
 }
```

Comment:
There's one small downside to this: The admin page also won't be registered in a later stage. Maybe it's better to wrap this in a callback on the `init` hook. Happy to change this part.

```suggestion
add_action( 'init', static function() {
	if ( is_admin() ) {
		 WP_Sentry_Admin_Page::get_instance();
	}
} );
```

We can't use `admin_init` because that's too late for `admin_menu`.